manifest.json

```
@@ -1,55 +1,56 @@
 {
     "name": "Galette",
     "id": "galette",
+    "packaging_format": 1,
+    "url": "https://www.galette.eu",
+    "license": "GPL-3.0-or-later",
+    "version": "0.9.4.2~ynh1",
+    "requirements": {
+        "yunohost": ">= 3.8"
```

```suggestion
        "yunohost": ">= 3.8.1"
```
We need to have the version 3.8.1 to remove the helpers on _common.sh